src/audio/copier/copier_dai.c

```diff
@@ -322,6 +322,9 @@ int copier_dai_create(struct comp_dev *dev, struct copier_data *cd,
 			return -EINVAL;
 		}
 		dai.out_fmt = &copier->out_fmt;
+#if CONFIG_COPIER_GAIN
+		dai.apply_gain = true;
+#endif
```

Comment:
What happens when no configuration is passed from the host (like in case of Linux driver)? Unity gain applied? This seems to be enabled by default.

Reply:
@kv2019i Yes, unity gain applied (gain = 1)